documentation/index.md

```diff
@@ -367,38 +388,45 @@ def pipe():
 debug_has_run = False
 pipe()
 assert debug_has_run == False
+## export RUN_DEBUG_NODES=True
+## debug_has_run = False
+## pipe()
+## assert debug_has_run == True
 ```
 
-## Advanced Usage
+## **Advanced Usage**
 
 ### Tag
-A _tag_ is a user defined identifier for an `ExecNode`. Every `ExecNode` can have a no _tag_ or multiple _tag_s.
+A _tag_ is a user defined identifier for an `ExecNode`. Every `ExecNode` is allowed to have None, one or multiple tags.
 
-You can tag an `ExecNode` with an `str`. You can also tag it using multiple tags (`Tuple[str]`)
+You can tag an `ExecNode` with an `str`. For multiple tags simply use a tuple (`Tuple[str]`).
 <!--pytest-codeblocks:cont-->
 
 ```python
-@xn(tag="twinkle toes")
+@xn(tag=("twinkle toes", "hot bird"))
 def a():
   print("I am tough")
 
-@xn
-def b():
+@xn(tag=("yummy jam"))
+def c():
   print("I am normal")
 
 @dag
 def pipeline():
   a()
-  b()
+  c()
 
 pipeline()
 xn_a, = pipeline.get_nodes_by_tag("twinkle toes")
+xn_b, = pipeline.get_nodes_by_tag("hot bird")
+xn_c, = pipeline.get_nodes_by_tag("yummy jam")
+assert xn_a == xn_b
 ```
 You can do whatever you want with this ExecNode:
 
 1. like looking in its arguments
-1. setting its priority
-1. changing it to become a debug `ExecNode`
+2. setting its priority
+3. changing it to become a debug `ExecNode`
```

Comment:
no need to write 1 2 and 3... markdown made it easier for use by letting us specify only 1.

This way we don't need to think about current number